public/data/2243421/info.json

```
@@ -0,0 +1,10 @@
+{
+    "nombre": "Daniel Arteaga",
+    "edad": 30,
+    "carrera": "Tegnología en desarrollo de software",
+    "semestre": 5,
+    "gustos": ["Audiovisual", "Deporte", "Cine", "Viajar"],
+    "noGustos": ["", "Comida picante"],
```

can you you remove that empty space?

```suggestion
    "noGustos": ["Comida picante"],
```